themes/default/layouts/learn/glossary/list.html

```diff
@@ -0,0 +1,48 @@
+{{ define "main" }}
+    <div class="container mx-auto flex items-start px-4 md:px-0">
+        <div class="md:w-3/12 my-8 sticky-sidebar md:pr-8">
+            <nav>
+
+            </nav>
+        </div>
+        <div class="md:w-7/12 my-8">
+            <header class="container mx-auto">
+                {{ partial "docs/breadcrumb.html" . }}
+                <h1>{{ .Title }}</h1>
+            </header>
+            <section class="container mx-auto my-8">
+                {{ .Content }}
+            </section>
+
+            {{ $terms := sort $.Site.Data.learn.glossary.entries "term" }}
+            {{ $page := .}}
+
+            {{ range $terms }}
+                {{ $anchor := .term | anchorize }}
+                <div>
+                    <ul>
+                        <li><a href="#{{ $anchor }}">{{ .term }}</a></li>
+                    </ul>
+                </div>
+            {{ end }}
```

Comment:
Little thing, but this results in a bunch of `ul`s stacked on top of one another, but we probably want one `ul` to wrap all the terms:

```suggestion
            <ul>
                {{ range $terms }}
                    {{ $anchor := .term | anchorize }}
                    <li><a href="#{{ $anchor }}">{{ .term }}</a></li>
                {{ end }}
            </ul>
```